test/db2.discover.test.js

```diff
@@ -238,6 +238,25 @@ describe('Discover model foreign keys', function() {
           }
         });
     });
+  it('should return an array of foreign keys for STRONGLOOP.RESERVATION',
+    function(done) {
+      db.discoverForeignKeys('RESERVATION', {
+        owner: config.schema,
+      },
+        function(err, models) {
+          if (err) {
+            console.error(err);
+            done(err);
+          } else {
+            models.forEach(function(m) {
+              // console.dir(m);
+              console.log(m);
+              assert(m.fkTableName === 'RESERVATION');
```

Comment:
We are just checking the foreign key table name. Can we check if it returns the correct foreign keys?